packages/ember-test-app/app/templates/cards.hbs

```diff
@@ -1,5 +1,15 @@
-<div data-theme="marketing" class="container grid p-3">
-  <Nrg::Mktg::Card class="g-col-12 g-col-md-4" />
-  <Nrg::Mktg::Card class="g-col-12 g-col-md-4" />
-  <Nrg::Mktg::Card class="g-col-12 g-col-md-4" />
+<div data-theme="marketing" class="container">
+  <p class="py-3">Row on desktop, stacked on mobile</p>
+  {{!-- template-lint-disable no-inline-styles --}}
+  <Nrg::Mktg::CardContainer class="bg-info rounded" style="--bs-gap: .5rem;" as |CardContainer|>
```

Comment:
Is the default gap too high? If so, we can change it in `@nrg-ui/css`.

Reply:
I'm not sure I have an opinion on that, I was more just aiming to give examples of how you would go about changing that if needed. Bootstrap doesn't have a gap class available, so I wanted to highlight that it has to be done through css. We can discuss that further if you'd like

Reply:
In that case, I might argue that we should add some `gap` classes, even if all they do is set the CSS variable.

Reply:
Should that be done after Keith's design system PR? Or go ahead and do that now?

Reply:
I think the embroider error was due to the promo component being moved and not being updated in its implementation in the test app. Should be good now!